src/app.js

```diff
@@ -26,6 +31,7 @@ application.register(fastifySwaggerUi, {
 });
 
 application.addSchema(doctorSchema);
+application.addSchema(userSchema);
```

Comment:
Lets add to schema folder index.js file and this file should import all schemas and export array with them. 
And in app.js import this array and in forEach add every schema